README.md

```diff
@@ -1,14 +1,21 @@
 ![The App Business](https://github.com/theappbusiness/MasterFastfile/blob/master/MasterFastfile.png)
 
 # MasterFastfile
- To setup your project to use the TAB master fastfile navigate to your project route and run the following command
+ To setup your project to use the TAB master fastfile navigate to your project root and run the following command
```

Comment:
Super finicky but I'd make this:
`To setup your project to use the TAB MasterFastfile, navigate to your project root and run the following command:`

But not an issue to leave as it.